src/Machine.js

```diff
@@ -222,6 +222,9 @@ class Machine {
       this._includeMacro(macro, context, buffer);
     } else {
       // source inclusion
+
+      // replace for compatibility with Windows
+      instruction.value = instruction.value.replace(/\\/g, "/");
```

Comment:
It looks like we can't do something like this, cose macro works only with include directive or in in-line mode, so we should write smth like @include @include s() + "filename", what, as far as I understand, is not supporting.